packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -734,6 +734,70 @@ export const components = ({
         disableRipple: true,
       },
     },
+    MuiCard: {
+      styleOverrides: {
+        root: () => ({
+          backgroundColor: odysseyTokens.HueNeutralWhite,
+          borderRadius: odysseyTokens.BorderRadiusOuter,
+          boxShadow: odysseyTokens.DepthMedium,
+          padding: odysseyTokens.Spacing5,
+          position: "relative",
+          transition: `all ${odysseyTokens.TransitionDurationMain} ${odysseyTokens.TransitionTimingMain}`,
+
+          "& img": {
+            height: "64px",
+          },
```

Comment:
Here's that `64px` again. Is this in our design tokens or somewhere we can add?

Reply:
Just added it as an exported const in `Tile.tsx`, so we can at least reference a single value in both the Tile and the components.tsx

Reply:
@jordankoschei-okta Do we need specific styling here if an SVG is passed in?